seminar-1/task_1.py

```diff
@@ -0,0 +1,13 @@
+#Найдите сумму цифр трехзначного числа.
+
+print("Введите трехзначное число")
+
+n = int (input())
```

Comment:
Не обязательно использовать функцию принц, чтобы вывести сообщение приглашение к вводу.
```suggestion
n = int (input("Введите трехзначное число "))
```